service/resource/deployment/worker_deployment.go

```diff
@@ -22,7 +22,7 @@ func newWorkerDeployments(customObject kvmtpr.CustomObject) ([]*extensionsv1.Dep
 	for i, workerNode := range customObject.Spec.Cluster.Workers {
 		capabilities := customObject.Spec.KVM.Workers[i]
 
-		cpuQuanity, err := key.CPUQuanity(capabilities)
+		cpuQuanity, err := key.CPUQuantity(capabilities)
```

Comment:
What about var name? It is also wrong

Reply:
```
for file in $(find . -type d \( -path ./.git -o -path ./vendor \)  -prune -o -print | grep '.*\.go'); do
    sed -i -e 's/Quanity/Quantity/g' $file
done
```

Reply:
Done, thanks :)

Reply:
You should really get a decent text editor. :trollface:

Reply:
My editor is capable for doing renaming, but it's too sloooow.